docs/en/integrations/data-ingestion/etl-tools/dbt/index.md

```diff
@@ -820,6 +820,26 @@ This process is shown below:
 
 <img src={require('./images/dbt_06.png').default} class="image" alt="lightweight delete incremental" style={{width: '100%'}}/>
 
+### insert_overwrite mode (Experimental)
+Performs the following steps:
+
+1. Create a staging (temporary) table with the same structure as the incremental model relation: CREATE TABLE {staging} AS {target}.
```

Comment:
Does it drop the temporary table before creating?

Reply:
Yes
https://github.com/ClickHouse/dbt-clickhouse/blob/46904a404ce7fa835d8c73a222729dbaef5372ed/dbt/include/clickhouse/macros/materializations/incremental/incremental.sql#L254